app/assets/stylesheets/style.css

```diff
@@ -49,6 +49,8 @@ h5,h6, h1 small, h3 small {
 .pl-content h3 {
   font-family: "Junction Light",lucida grande,lucida sans console,sans-serif;
   padding-top: 0.4em;
+  overflow-wrap: break-word;
```

Comment:
Him it looks like this may be part of the h3 style, if I read it right. Is it possible to move it to just a plain .pl-content style, so it affects the entire pl-content container? 

If you like I could make a code suggestion to show how?

Thank you!

Reply:
should i create a .pl-content style so cause all the ones on the stylesheet have either 
a header or paragraph attached to it